libraries/AP_Notify/AP_Notify.cpp

```diff
@@ -221,15 +221,20 @@ void AP_Notify::add_backends(void)
   #endif
 
 #elif CONFIG_HAL_BOARD == HAL_BOARD_CHIBIOS
-#ifdef HAL_HAVE_PIXRACER_LED
+# ifdef HAL_HAVE_PIXRACER_LED
     ADD_BACKEND(new PixRacerLED());
-#else
+# else
     ADD_BACKEND(new AP_BoardLED());
-#endif
+# endif
     ADD_BACKEND(new ToshibaLED_I2C(TOSHIBA_LED_I2C_BUS_EXTERNAL));
     ADD_BACKEND(new ToshibaLED_I2C(TOSHIBA_LED_I2C_BUS_INTERNAL));
     ADD_BACKEND(new Display());
-#lese
+#elif CONFIG_HAL_BOARD == HAL_BOARD_F4LIGHT
+    ADD_BACKEND(new ToshibaLED_I2C(TOSHIBA_LED_I2C_BUS_EXTERNAL));
+    ADD_BACKEND(new Display());
+    ADD_BACKEND(new Buzzer());
+//    ADD_BACKEND(new AP_BoardLED2()); // needs AP_BoardLED2 in master
```

Comment:
normally we don't leave commented out code in there so probably best to remove this line.

Reply:
this commented code waits for #7585, so after it will be merged this becomes just a reminder